src/widgets/CompanyMainControlPanel/ui/CompanyForm.tsx

```diff
@@ -35,32 +42,32 @@ export const CompanyForm: FC = observer(() => {
 
   // todo Тоже сверху передавать
   const [companyUsersStore, setCompanyUsersStore] = useState<CompanyUsersModel>(
-    userStores.getCustomCompanyUserStore(Number(companyId)),
+    initialCompanyUsersStore,
   );
 
   useEffect(() => {
     try {
-      const company = companiesStore.getCompany(Number(companyId));
+      const company = companiesStore.getCompany(Number(currentCompanyId));
```

Comment:
```suggestion
      const company = companiesStore.getCompany(+currentCompanyId);
```
Преобразование к числу лучше делать так